src/apps/engine/src/compiler.cpp

```diff
@@ -1583,6 +1632,10 @@ bool COMPILER::Compile(SEGMENT_DESC &Segment, char *pInternalCode, uint32_t pInt
                                             return false;
                                         }
                                         real_var->value->Set(1);
```

Comment:
Can we extract kind of `SetVariable` function here accepting Varinfo and value, like this:
```
void Compiler::SetVariable(VarInfo * var, const auto& value) {
  var->value->Set(value);
  if(use_script_cache_) {
    script_cache_.variables.back().value->Set(value);
  }
]
```
in order to avoid much code bloating?

Reply:
moved assignment in one spot